main.swift

```diff
@@ -263,7 +266,7 @@ if let organizationName = organizationName {
 
 print("---------------------------------------------------------------------")
 
-if !askForBooleanInfo(question: "Proceed? ✅") {
+if !arguments.isForce && !askForBooleanInfo(question: "Proceed? ✅") {
```

Comment:
How about nesting these if statements instead, makes it clear that `forceEnabled` is a precondition for `askForBooleanInfo()`.